SoftLayer/CLI/bandwidth/pools_create.py

```diff
@@ -21,13 +21,36 @@
     "FRA": "FRA"
 }
 
+regions = ['SJC/DAL/WDC/TOR/MON', 'AMS/LON/MAD/PAR', 'SNG/HKG/OSA/TOK', 'SYD', 'MEX', 'SAO', 'CHE', 'MIL', 'SEO', 'FRA']
+HELP_STMT = "Ex: 'HKG' or 'SNG/HKG/OSA/TOK'"
+
+
+def check_region_param(ctx, param, value):
```

Comment:
```suggestion
def check_region_param(ctx, param, value):  # pylint: disable=unused-argument
```
this disables the error about unused params, so you can remove the `_ = [ctx, param]` line.